app/scripts/controllers/overview.js

```diff
@@ -692,10 +697,82 @@ function OverviewController($scope,
     return true;
   };
 
+  // part of lodash since 4.0.0
+  function byKey(selector) {
+    return function (accum, item) {
+      var key = selector(item)
+      accum[key] = item;
+      return accum;
+    }
+  }
+
+  function getOvmId(vm) {
+    var ownerReferences = vm.metadata.ownerReferences;
+    if (!ownerReferences) {
+      return null;
+    }
+    return _(ownerReferences)
+      .filter(function (reference) {
+        return reference.kind === 'OfflineVirtualMachine';
+      })
+      .map(function (reference) {
+        return reference.uid;
+      })
+      .first();
+  }
+
+  function updateVirtualMachineMapping() {
+    if (!overview.offlineVirtualMachines || !overview.virtualMachines || !overview.pods) {
+      return;
+    }
+    var vmIdToPod = _(overview.pods)
+      .values()
+      .filter(function (pod) {
+        return !!_.get(pod, 'metadata.labels["kubevirt.io/vmUID"]')
+      })
+      .reduce(byKey(function (pod) {
+        return pod.metadata.labels['kubevirt.io/vmUID'];
+      }), {});
+    var ovmIdToVm = _(overview.virtualMachines)
+      .values()
+      .filter(function (vm) {
+        return !!getOvmId(vm)
+      })
+      .reduce(byKey(function (vm) {
+        return getOvmId(vm);
+      }), {});
+    var podsOfOvms = [];
+    _.forEach(overview.offlineVirtualMachines, function (ovm) {
```

Comment:
nit: we try to always use `_.each` for consistency (so we're not using `_.forEach` in some places and `_.each` in others)

Reply:
done